RevenueCatUI/Views/RemoteImage.swift

```diff
@@ -19,42 +19,80 @@ struct RemoteImage: View {
     let url: URL
     let aspectRatio: CGFloat?
     let maxWidth: CGFloat?
+    let lowResSuffix: String?
+
+    var fetchLowRes: Bool {
+        return false
+        // Replace `return false` with the following when the back end is ready
+        // lowResSuffix != nil
+    }
+
+    @StateObject
+    private var highResLoader: ImageLoader = .init()
 
     @StateObject
-    private var loader: ImageLoader = .init()
+    private var lowResLoader: ImageLoader = .init()
 
-    init(url: URL, aspectRatio: CGFloat? = nil, maxWidth: CGFloat? = nil) {
+    init(url: URL, aspectRatio: CGFloat? = nil, maxWidth: CGFloat? = nil, lowResSuffix: String? = "_low_res") {
         self.url = url
         self.aspectRatio = aspectRatio
         self.maxWidth = maxWidth
+        self.lowResSuffix = lowResSuffix
     }
 
     var body: some View {
         Group {
-            switch self.loader.result {
-            case .none:
-                self.emptyView(error: nil)
-
-            case let .success(image):
-                if let aspectRatio {
-                    image
-                        .fitToAspect(aspectRatio, contentMode: .fill)
-                        .frame(maxWidth: self.maxWidth)
-                        .accessibilityHidden(true)
-
+            if case let .success(image) = highResLoader.result {
+                displayImage(image)
+            } else if case let .success(image) = lowResLoader.result {
+                displayImage(image)
+            } else if case let .failure(highResError) = highResLoader.result {
+                if !fetchLowRes {
+                    emptyView(error: highResError)
+                } else if case .failure = lowResLoader.result {
```

Comment:
I think we're ignoring the low res error. Should we log something in this case?

Reply:
Good point, but ImageLoader logs when it hits an error (below), so I think we are ok here.

```
    @Published
    private(set) var result: Value? {
        didSet {
            if let result {
                Logger.verbose(Strings.image_result(result.map { _ in () }))
            }
        }
    }
    ```

Reply:
Ahh then this is fine, thanks!